.travis.yml

```diff
@@ -8,4 +8,9 @@ env:
   global:
   - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
 before_script:
-  - jdk_switcher use oraclejdk8
\ No newline at end of file
+  - jdk_switcher use oraclejdk8
+  - gdalinfo --version
+addons:
+  apt:
+    packages:
+    - gdal-bin
```

Comment:
Can you add a line to the README about installing this?  `brew install gdal` worked for me on MacOSX.